GrandCentralBoard/Widgets/Watch/WatchWidget.swift

```diff
@@ -6,9 +6,7 @@
 import UIKit
 import GrandCentralBoardCore
 
-
-private let slideshowInterval = 10.0
-private let secondsInDay: NSTimeInterval = 3600
+private let secondsInAnHour: NSTimeInterval = 3600
```

Comment:
I can't agree, properties should always start with a lowercase letter. As depicted in these guidelines: https://github.com/raywenderlich/swift-style-guide#naming, or here https://github.com/github/swift-style-guide#always-specify-access-control-explicitly-for-top-level-definitions . Also I know SwiftLint would argue.